app/Schema/Postgres.php

```diff
@@ -6,7 +6,12 @@
 use Kanboard\Core\Security\Token;
 use Kanboard\Core\Security\Role;
 
-const VERSION = 90;
+const VERSION = 91;
+
+function version_91(PDO $pdo)
+{
+    $pdo->exec("ALTER TABLE columns ADD COLUMN hide_in_dashboard INTEGER DEFAULT 0 NOT NULL");
```

Comment:
For Postgres, it's probably better to use the type `BOOLEAN` instead (same as other flags). If you can make this change, I'm good to merge.